concordium-sdk/src/main/java/com/concordium/sdk/responses/transactionstatus/RejectReasonNonExistentCredIDs.java

```diff
@@ -1,20 +1,31 @@
 package com.concordium.sdk.responses.transactionstatus;
 
+import com.concordium.sdk.transactions.CredentialRegistrationId;
 import com.fasterxml.jackson.annotation.JsonCreator;
 import com.fasterxml.jackson.annotation.JsonProperty;
 import lombok.Getter;
 import lombok.ToString;
 
+import java.util.ArrayList;
 import java.util.List;
+import java.util.Objects;
 
+/**
+ * A credential id that was to be removed is not part of the account.
+ */
 @ToString
 public class RejectReasonNonExistentCredIDs extends RejectReason {
     @Getter
-    private final List<String> ids;
+    private List<CredentialRegistrationId> ids;
 
     @JsonCreator
     RejectReasonNonExistentCredIDs(@JsonProperty("contents") List<String> ids) {
-        this.ids = ids;
+        if (!Objects.isNull(ids)) {
+            this.ids = new ArrayList<>();
+            for (String id : ids) {
+                this.ids.add(CredentialRegistrationId.from(id));
+            }
+        }
```

Comment:
Same as the comments above. I don't get this null handling. Empty lists are great.